db/cyndi_setup_devel.sql

```diff
@@ -19,14 +19,16 @@ SELECT
     stale_timestamp + INTERVAL '1' DAY * 7 AS stale_warning_timestamp,
     stale_timestamp + INTERVAL '1' DAY * 14 AS culled_timestamp,
     tags,
-    system_profile_facts as system_profile
-FROM dblink('dbname=insights user=insights', 'select id, account, display_name, created_on, modified_on, stale_timestamp, tags, system_profile_facts from hosts') as hosts(
+    system_profile_facts as system_profile,
+    canonical_facts->'insights_id' as insights_id
+FROM dblink('dbname=insights user=insights', 'select id, account, display_name, created_on, modified_on, stale_timestamp, tags, system_profile_facts, canonical_facts from hosts') as hosts(
```

Comment:
This is meant to simply replicate the view we are given (and have no control over) from Cyndi. Here is some code I found within their operator that sets up the view: https://github.com/RedHatInsights/cyndi-operator/blob/611d362fc29c9b8c391d0081c6686c79c2b1736a/controllers/database/app_database.go#L17

Reply:
I don't understand the problem here, I consulted this with someone from the inventory team. The `insights_id` field is coming from the `canonical_facts` jsonb column and it's available in the view according to the docs.
```
insights=# \c insights;
psql (11.12 (Debian 11.12-0+deb10u1), server 13.4 (Debian 13.4-1.pgdg100+1))
WARNING: psql major version 11, server major version 13.
         Some psql features might not work.
You are now connected to database "insights" as user "insights".
insights=# SELECT canonical_facts->'insights_id' FROM hosts;
                ?column?                
----------------------------------------
 "e26aea48-56d4-43c0-823f-6ac5c09ba63a"
(1 row)
```

Reply:
Sorry, yes - I see the insights_id is actually part of the view we are given after all. We should be ok.